File: seed/go-sdk/path-parameters/inline-path-parameters/dynamic-snippets/example4/snippet.go
package example

import (
    client "github.com/fern-api/path-parameters-go/client"
    context "context"
    path "github.com/fern-api/path-parameters-go"
)

func do() () {
    client := client.NewClient()
    client.User.SearchOrganizations(
        context.TODO(),
        "organizationId",
        &path.SearchOrganizationsRequest{
            Limit: path.Int(
                1,
            ),
        },
    )
}

File: seed/ts-sdk/multi-line-docs/src/dynamic-snippets/example1/snippet.ts
import { SeedMultiLineDocsClient } from "../..";

async function main(): Promise<void> {
    const client = new SeedMultiLineDocsClient({
        environment: "https://api.fern.com",
    });
    
    await client.user.createUser({
        name: "name",
        age: 1,
    });
}
main();
